### ISSUE TITLE
Support Excluding Populations Based on Previous Work

### ISSUE THREAD

**Issue by LeanderK** [coolcrowd/control-ui#10]
currently not supported by mturk! The database has special populations based on previous work, these can be used to exclude them from working on an experiment. (currently also not in spec)

**Comment by kelunik** [comment]
It's currently implemented with the `worker_quality_threshold` field.

How should something like this be specified?

**Comment by LeanderK** [comment]
no, based on populations. Each experiment has a population attached, which got automatically generated. These can be used to exclude workers based on previous work

**Comment by kelunik** [comment]
So you want to restrict experiments to workers which have also have participated in experiment XY?
